src/snyk/common/views/issueTreeProvider.ts

```diff
@@ -35,6 +38,14 @@ export abstract class ProductIssueTreeProvider<T> extends AnalysisTreeNodeProvid
     );
   }
 
+  getAIFix(issue: Issue<T>): boolean {
+    const myIssue = issue as unknown as Issue<CodeIssueData>;
+    if (myIssue.additionalData.hasAIFix) {
+      return true;
+    }
+    return false;
```

Comment:
this seems to be dead code, its re-implemented in the snykCode/issueTreeProvider